.github/CODEOWNERS

```
@@ -136,6 +136,7 @@ receiver/awscontainerinsightreceiver/                @open-telemetry/collector-c
 receiver/awsecscontainermetricsreceiver/             @open-telemetry/collector-contrib-approvers @Aneurysm9
 receiver/awsfirehosereceiver/                        @open-telemetry/collector-contrib-approvers @Aneurysm9
 receiver/awsxrayreceiver/                            @open-telemetry/collector-contrib-approvers @willarmiros
+receiver/azureblobreceiver/                          @open-telemetry/collector-contrib-approvers @eedorenko @pmm-sumo
```

Przemek is no longer involved in the project:

```suggestion
receiver/azureblobreceiver/                          @open-telemetry/collector-contrib-approvers @eedorenko @mx-psi
```